core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -52,13 +50,6 @@ object SubmissionMonitorActor {
    */
   case class ExecutionServiceStatusResponse(statusResponse: Seq[Try[Option[(WorkflowRecord, Option[ExecutionServiceOutputs])]]]) extends SubmissionMonitorMessage
   case class StatusCheckComplete(terminateActor: Boolean) extends SubmissionMonitorMessage
-
-  /**
```

Comment:
I moved this gauge stuff out of the `SubmissionMonitorActor` and into the `SubmissionSupervisor`. The reason is that `SubmissionMonitorActor` is transient -- it only exists while the submission is running. The gauge should be able to monitor terminal and non-terminal submissions/workflows, so the supervisor seemed like a good fit.